### PR TITLE
Allow multiple deadlines to be added with descriptions associated

### PR DESCRIPTION
**Pull request by tehkokhoe**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -107,10 +109,36 @@ public static Email parseEmail(String email) throws ParseException {
...
-        if (!Deadline.isValidDeadline(trimmedDeadline)) {
+        StringBuilder originalInput = new StringBuilder(trimmedDeadline);
+        originalInput.reverse();
+        String[] reversedAndSplit = originalInput.toString().split("\\s+", 2);
+        if (reversedAndSplit.length < 2) {
             throw new ParseException(Deadline.MESSAGE_CONSTRAINTS);
         }
-        return new Deadline(trimmedDeadline);
+        String description = new StringBuilder(reversedAndSplit[1]).reverse().toString();
+        String date = new StringBuilder(reversedAndSplit[0]).reverse().toString();
```
Comment (weijuey) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:119]: Sorry, could you explain why did you choose to reverse the strings before calling split? I find that there's no issue calling split right away :thinking:

**Review by weijuey — COMMENTED** [review]
This turned out to be a bigger change than I thought, gj getting it out for v1.2 :+1: . Have a few comments, but otherwise LGTM!

File: src/main/java/seedu/address/model/person/Person.java [hunk 2]
```diff
@@ -135,14 +133,19 @@ public String toString() {
...
-                .append(getDeadline())
-                .append("; Notes: ")
-                .append(getNotes());
+                .append(getAddress());
+        ArrayList<Deadline> deadlines = getDeadlines().getDeadlines();
+        builder.append("; Deadline(s): ");
+
+        for (Deadline deadline : deadlines) {
+            builder.append(deadline);
+        }
```
[the remark this answers is not in the document]
Comment (tehkokhoe) [reply src/main/java/seedu/address/model/person/Person.java:142]: OK I can do this

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 1 above)
Comment (tehkokhoe) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java:119]: I was thinking if someone inputs a description including whitespace like, "deadline 1 d/read every single book 1/1/2023", I only want to split the string on the last delimiter. So I reverse the string, split the string on first delimiter then reverse the string again.

Comment (weijuey) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java:119]: Ah I see, I didn't think of that, thanks for explaining! :smile:

**Pull request merged**